### PR TITLE
ENDOC-170 6.3 Release Notes

### PR DESCRIPTION
**Pull request by nshaw**

### CODE REVIEW

File: vuepress/docs/v6.3/docs/releases/README.md [hunk 1]
```diff
@@ -1,100 +1,157 @@
...
+    * For more information on the CLI see [this page](../reference/entando-cli.md).
 
-* Included a set of additional default widgets, content types, content templates, and page templates
+* Add compatibility fixes when running Entando tools on Linux, MacOS, and Windows 10.
 
-### Entando Component Generator
+### Quickstart Installs
 
-* Improved Microsoft Windows Support (specifically Windows 10 Professional)
+* Enable the one step HTTP installer for use on Linux, MacOS, and Windows 10 to setup a fully functional Entando Application in Kubernetes.
```
Comment (es-entando) [inline vuepress/docs/v6.3/docs/releases/README.md:128]: "set up"

**Review by es-entando — APPROVED** [review]

File: vuepress/docs/v6.3/docs/releases/README.md [hunk 2]
```diff
@@ -1,100 +1,157 @@
...
+* Remove support for weak encryption algorithms, including MD5, Argon, 3DES
+
+* Update to recent and more secure versions of spring-boot, Apache CXF, Keycloak engine and client libraries
+
+* Improve log sanitization
 
 ### Entando Component Repository (ECR)
 
-* ECR now relies only on GIT repositories for the distribution of bundles
+* Upgrade to Java 11 for all Entando Bundles
```
[the remark this answers is not in the document]
Comment (nshaw) [reply vuepress/docs/v6.3/docs/releases/README.md:78]: I'll refine the language a bit but they are distinct items, core Entando images vs blueprint-generated ECR bundles, so I'll keep both.

File: vuepress/docs/v6.3/docs/releases/README.md [hunk 3]
```diff
@@ -1,100 +1,157 @@
...
+
+* Rename *entando-bundle-cli* repository and tool to *entando-bundler* (or just *bundler*) to avoid confusion with the new *entando-cli* command line tool
+
+* Add capability to export all content in Entando Application via the *entando-bundler*
+
+    * This does not include users but does include groups and permissions.
+
+    * Some content situations (e.g. widget’s css pointing to a root resource) may require the exported bundle to be manually adjusted before importing to another Entando Application
```
[the remark this answers is not in the document]
Comment (nshaw) [reply vuepress/docs/v6.3/docs/releases/README.md:86]: Alternate is good, thanks. The root resource has to do with the way bundle paths are dealt with. It's just a limitation in the current design/tool.

File: vuepress/docs/v6.3/docs/releases/README.md [hunk 4]
```diff
@@ -1,100 +1,157 @@
...
+* Improve log sanitization
 
 ### Entando Component Repository (ECR)
 
-* ECR now relies only on GIT repositories for the distribution of bundles
+* Upgrade to Java 11 for all Entando Bundles
+
+* Rename *entando-bundle-cli* repository and tool to *entando-bundler* (or just *bundler*) to avoid confusion with the new *entando-cli* command line tool
+
+* Add capability to export all content in Entando Application via the *entando-bundler*
```
[the remark this answers is not in the document]
Comment (nshaw) [reply vuepress/docs/v6.3/docs/releases/README.md:82]: Got it

File: vuepress/docs/v6.3/docs/releases/README.md [hunk 5]
```diff
@@ -1,100 +1,157 @@
...
+
+    * This does not include users but does include groups and permissions.
+
+    * Some content situations (e.g. widget’s css pointing to a root resource) may require the exported bundle to be manually adjusted before importing to another Entando Application
+
+* Add a *forced overwrite* installation strategy for Entando Bundles. This strategy creates new objects or updates objects already existing on the target system in order to allow business cases like:
```
[the remark this answers is not in the document]
Comment (nshaw) [reply vuepress/docs/v6.3/docs/releases/README.md:88]: good call, past release notes were all past tense. Will fix.

File: vuepress/docs/v6.3/docs/releases/README.md [hunk 6]
```diff
@@ -1,100 +1,157 @@
...
+* Improve the bundle plugin descriptor to provide new capabilities. New properties:
 
-* Enabled support for bundles with multiple versions
+    * 1) *deploymentBaseName*: The base name to assign to the Kubernetes pods. If not present the base name will be generated from the docker organization, image name and image version. If the generated name is too long it will be truncated in order to respect Kubernetes constraints.
 
-* Reliability improvements in the installation/uninstallation process, e.g. ability to install/uninstall a specific bundle version
+    * 2) *ingressPath*: the ingress path to assign to the plugin deployment
 
-* Fixes and improvements to the web interface
+    * 3) *permissions*: a list of keycloak clientId / role to bind each to the other
```
[the remark this answers is not in the document]
Comment (nshaw) [reply vuepress/docs/v6.3/docs/releases/README.md:104]: Got it

File: vuepress/docs/v6.3/docs/releases/README.md [hunk 7]
```diff
@@ -1,100 +1,157 @@
...
+
+* Add a *forced overwrite* installation strategy for Entando Bundles. This strategy creates new objects or updates objects already existing on the target system in order to allow business cases like:
+
+    * 1) continuous deployment in development scenarios
+
+    * 2) continuous deployment of staging to production
+
+    * 3) migration of Entando Applications. The developer/admin will still be responsible for the update of data structures, as appropriate.
+
+    * 4) develop a component, export as a bundle, then import to a test instance for QA process, then import into production instance
```
[the remark this answers is not in the document]
Comment (nshaw) [reply vuepress/docs/v6.3/docs/releases/README.md:96]: got it

File: vuepress/docs/v6.3/docs/releases/README.md [hunk 8]
```diff
@@ -1,100 +1,157 @@
...
+    * 3) *permissions*: a list of keycloak clientId / role to bind each to the other
 
-* Updated documentation, e.g. common use cases, uninstall flow, CRDs, ingresses and a troubleshooting guide
+### Developer Experience
 
-* Temporarily disabled support for Pages in bundles (introduced with 6.1) due to problems during uninstallation
+* Entando 6.3 now includes a Command Line Interface, *entando-cli* or simply *ent*, intended to accelerate Entando development by automating common development tasks.
 
-* Added full support for composite CMS attributes
+    * Simplify the quickstart install process via a one line script leveraging a multipass VM or for direct install via *ent quickstart*
```
[the remark this answers is not in the document]
Comment (nshaw) [reply vuepress/docs/v6.3/docs/releases/README.md:110]: Got it

File: vuepress/docs/v6.3/docs/releases/README.md (hunk 1 above)
Comment (nshaw) [reply vuepress/docs/v6.3/docs/releases/README.md:128]: Got it

**Pull request merged**